verification/curator-service/ui/src/components/new-case-form-fields/LocationForm.tsx

```diff
@@ -122,7 +128,7 @@ export function PlacesAutocomplete(
                 setInputValue(newInputValue);
             }}
             renderInput={(params): JSX.Element => (
-                <div>
+                <>
```

Comment:
Interesting. This can be used to group the fields into one Element to return?

Reply:
yes, I've been using it plenty, super useful :)